sessions-to-ics.js

```diff
@@ -90,21 +91,19 @@ function toIcs(sessionContainer) {
         duration: timeDetails.duration,
         location: sessionProps["Location"],
         title: title,
-        description: `${sessionType}\n\n${description}`,
-        sessionType: sessionType
+        description: `${sessionType}\n\n${description}`
     };
-    return event;
+    return { sessionType, event };
 }
 
 const events = {};
 $(".awsui-util-mb-xl").each((idx, row) => {
-    const event = toIcs(row);
+    const { sessionType, event } = toIcs(row);
```

Comment:
@francoroy thanks for the cleanup! That's much nicer than my "retrieve and delete."